benchmarks/transactions-generator/src/lib.rs

```diff
@@ -0,0 +1,207 @@
+        let view_client_sender = self.view_client_sender.clone();
+
+        if self.params.tps == 0 {
+            anyhow::bail!("target TPS should be > 0");
+        }
+        let mut stats = Stats { pool_accepted: 0, pool_rejected: 0, processed: 0, failed: 0 };
+        let mut stats_prev = stats.clone();
+        let mut tx_interval =
+            tokio::time::interval(Duration::from_micros(1_000_000 / self.params.tps));
+        let mut block_interval = tokio::time::interval(Duration::from_secs(5));
+        let mut report_interval = tokio::time::interval(Duration::from_secs(1));
+
+        let handle = tokio::spawn(async move {
+            let mut rnd: StdRng = SeedableRng::from_entropy();
+            let mut block_hash = CryptoHash::default();
+            tracing::debug!(target: "transaction-generator", "tx generation loop starting");
+            loop {
+                tokio::select! {
+                    _ = tx_interval.tick() => {
+                        Self::generate_send_transaction(
+                            &mut rnd,
+                            &mut accounts,
+                            &block_hash,
+                            &client_sender,
+                            &mut stats).await;
+                    }
+                    _ = block_interval.tick() => {
+                        if let Ok(Ok(block_view)) = view_client_sender.block_request_sender.send_async( GetBlock(BlockReference::latest())).await {
+                             block_hash = block_view.header.hash;
+                             tracing::trace!(target: "transaction-generator",
+                                 block_hash=format!("{block_hash:?}"), "update");
+                        } else {
+                            tracing::warn!(target: "transaction-generator", "block_hash update failed");
+                        }
+                    }
+                    _ = report_interval.tick() => {
+                        stats.processed = TRANSACTION_PROCESSED_SUCCESSFULLY_TOTAL.get();
+                        stats.failed = TRANSACTION_PROCESSED_FAILED_TOTAL.get();
+                        tracing::info!(target: "transaction-generator",
+                            total=format!("{stats:?}"),);
+                        tracing::info!(target: "transaction-generator",
+                            diff=format!("{:?}", stats.clone() - stats_prev),);
+                        stats_prev = stats.clone();
+                    }
+                }
+            }
+        });
+
+        self.runner = Some(handle);
+        Ok(())
+    }
+
+    /// Generates a transaction between two random (but different) accounts and pushes it to the `client_sender`
+    async fn generate_send_transaction(
+        rnd: &mut StdRng,
+        accounts: &mut [account::Account],
+        block_hash: &CryptoHash,
+        client_sender: &ClientSender,
+        stats: &mut Stats,
+    ) {
+        const AMOUNT: near_primitives::types::Balance = 1_000;
```

Comment:
nit: Why send more than 1? The higher `AMOUNT`, the sooner an account may run out of funds in long running benchmarks.